taskcluster/scripts/bitrise-schedule.py

```diff
@@ -165,6 +173,8 @@ async def download_log(client, build_slug):
 
     response = await do_http_request_json(client, url)
     download_url = response["expiring_raw_log_url"]
+    if not download_url:
+        raise TaskException("Bitrise has no log to offer for job {}. Please check https://app.bitrise.io/app/{}".format(build_slug, BITRISE_APP_SLUG_ID))
```

Comment:
I've come across issues where logs aren't exposed by the REST API, but are visible on their webUI. I made this error message clearer.